tests/acceptance/stepDefinitions/sharingContext.js

```diff
@@ -510,6 +529,7 @@ Given(
       null,
       { password: password }
     )
+    await updateTimeOfLastShareOperation()
```

Comment:
add this line in `shareFilefolder()` function as there are other steps too that do share.